tests/unit/validation/test_rules.py

```diff
@@ -143,6 +143,19 @@ def test_all_between_or_at():
     assert len(errors) == 1
     assert NotBetweenOrAtError("test", "value", [1, 4, 5, 6], (0.2, -0.2)) in errors
 
+    int8_nan = power_grid_meta_data["input"]["transformer"].nans["tap_pos"]
```

Comment:
maybe rename this to just

```suggestion
    nan_value = power_grid_meta_data["input"]["transformer"].nans["tap_pos"]
```